docs/middleware.rst

```diff
@@ -27,6 +27,21 @@ several built-in middlewares whose list is:
 - ``UserInterfaceMiddleware``: adds some user-related information like the client
   IP address to the event.
 
+There are also some "special" middlewares that should be executed after all the
+other middlewares so that they can sanitize and remove sensitive information before
+they reach the Sentry server (not all of them are enabled by default):
```

Comment:
Should we mark which one are enabled by default?

Reply:
Yes it's a good idea, done!